tests/query-node/run-tests.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -e
+
+SCRIPT_PATH="$(dirname "${BASH_SOURCE[0]}")"
+cd $SCRIPT_PATH
+mkdir -p hydra-test/
+cd hydra-test
+
+npx @dzlzv/hydra-cli scaffold --wsProviderUrl=ws://joystream-node:9944/ --projectName=Test
```

Comment:
@dzhelezov should this initial execution of the scaffold command be specific version of hydra-cli perhaps?

Is there a way to include the version in the package name when calling npx? like:  `npx @dzlzv/hydra-cli:v0.0.14 ...`

Reply:
ah i think its like this:
`npx @dzlzv/hydra-cli@v0.0.14`

Reply:
Actually, the scaffold here is going to be used only to 1) create .env file 2) create Dockerfile 3) create a docker-compose file. Perhaps we can replace this dependency altogether and use fixed files?

Reply:
The actual dependencies are all hidden in the hydra-cli docker image in Dockerfile.hydra